File: app-kit/src/main/java/media/pixi/appkit/domain/chats/MessageSendStatus.java
package media.pixi.appkit.domain.chats;


public enum MessageSendStatus {
    None,
    Created,
    WillUpload,
    Uploading,
    DidUpload,
    WillSend,
    Sending,
    Sent,
    Delivered,
    Failed,
}

File: app-kit/src/main/java/media/pixi/appkit/domain/chats/models/MessageSendStatus.kt
package media.pixi.appkit.domain.chats.models


enum class MessageSendStatus {
    None,
    Created,
    WillUpload,
    Uploading,
    DidUpload,
    WillSend,
    Sending,
    Sent,
    Delivered,
    Failed
}
